src/utils/paymentPlanStrings.tsx

```diff
@@ -6,16 +6,25 @@ import { priceFromCents } from 'utils'
 
 export const paymentPlanShorthandName = (payment: EligibilityPlan): ReactNode => {
   const { deferred_days, deferred_months, installments_count: installmentsCount } = payment
-  const deferredDaysCount = deferred_days + deferred_months * 30
-
-  if (installmentsCount === 1 && deferredDaysCount > 0) {
-    // For now there is no distinction between deferred days and deferred months, it will be J+ anyway
+  
+  if (deferred_days) {
```

Comment:
I think we should keep the 1st part of the if here, like 
```suggestion
  if (installmentsCount === 1 && deferred_days) {
```

and same for the month : 
```suggestion
  if (installmentsCount === 1 && deferred_months) {
```

It is just because, for the corner case (that is  not really used, but we never know) where a plan is for example P3X + 15 J,
We would like to stay consistent with what was displayed before.
For deferred + P3X, the button will still display only `3x` for example, and then the installments card will show that the 1st installment will not be `today` but `in xx days/months`.

With the `if` that you made, a deferred + PNX button will be displayed as a `J/M + xx`, it also make sense, but I feel like it's better not to change the behavior for this tricky case, what do you think ?